src/Command.php

```diff
@@ -146,6 +148,64 @@ function ( $plugin ) use ( $search ) {
 		return null;
 	}
 
+	/**
+	 * Given a Plugin Object, we determine what is the version storage file path.
+	 *
+	 * @since TBD
+	 *
+	 * @param mixed $plugin An object of the plugin, based on tut.json structure.
+	 *
+	 * @return null|string
```

Comment:
Why null and not an empty string?

Reply:
I wanted to leave room for the diff in value. Like empty would be something else in the future